lib/getLatestRelease.js

```diff
@@ -0,0 +1,100 @@
+import includes from 'lodash/includes';
+import async from 'async';
+import GitHubApi from 'github';
+
+var github = new GitHubApi();
+
+if (process.env.GH_TOKEN) {
+  github.authenticate({
+    type: 'token',
+    token: process.env.GH_TOKEN
+  });
+}
```

Comment:
Is there / does there need to be any error-handling for if `GH_TOKEN` either isn't set, or is invalid?